src/components/Header/index.jsx

```diff
@@ -0,0 +1,136 @@
+import React, { useEffect, useRef, useState } from "react";
+import { data } from "../../data/test";
+import Box from "./Boxinput";
+import BoxinputForcus from "./BoxinputForcus";
+import "./style.scss";
+import {
+  faArrowLeft,
+  faArrowRight,
+  faArrowUp,
+  faCancel,
+  faDiamond,
+  faDownload,
+  faGroupArrowsRotate,
+  faIndustry,
+  faSearch,
+  faTShirt,
+  faUser,
+} from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+Headers.propTypes = {};
+
+function Headers(props) {
+  const [text, setText] = useState("");
+  const [box, setBox] = useState(false);
```

Comment:
Nếu là boolean thì nên đặt tên có chữ 'is' phía trước thì nó sẽ trông giống một biến điều kiện hơn
```suggestion
const [isBox, setIsBox] = useState(false);
```